zookeeper-server/src/test/java/org/apache/zookeeper/test/X509AuthTest.java

```diff
@@ -96,16 +96,16 @@ public void testUntrustedAuth() {
     public void testSANBasedAuth() {
         // Set JVM properties to enable SAN-based client id extraction
         System.setProperty(
-            X509AuthenticationProvider.ZOOKEEPER_X509AUTHENTICATIONPROVIDER_CLIENTCERTIDTYPE,
+            X509AuthenticationProvider.ZOOKEEPER_X509AUTHENTICATIONPROVIDER_CLIENT_CERT_ID_TYPE,
             "SAN");
         System.setProperty(
-            X509AuthenticationProvider.ZOOKEEPER_X509AUTHENTICATIONPROVIDER_CLIENTCERTIDSANMATCHTYPE,
+            X509AuthenticationProvider.ZOOKEEPER_X509AUTHENTICATIONPROVIDER_CLIENT_CERT_ID_SAN_MATCH_TYPE,
             "6");
         System.setProperty(
-            X509AuthenticationProvider.ZOOKEEPER_X509AUTHENTICATIONPROVIDER_CLIENTCERTIDSANMATCHREGEX,
+            X509AuthenticationProvider.ZOOKEEPER_X509AUTHENTICATIONPROVIDER_CLIENT_CERT_ID_SAN_MATCH_REGEX,
             TestCertificate.TEST_SAN_STR);
         System.setProperty(
-            X509AuthenticationProvider.ZOOKEEPER_X509AUTHENTICATIONPROVIDER_CLIENTCERTIDSANEXTRACTREGEX,
+            X509AuthenticationProvider.ZOOKEEPER_X509AUTHENTICATIONPROVIDER_CLIENT_CERT_ID_SAN_EXTRACT_REGEX,
```

Comment:
Can we test this with more real word examples with capturing groups instead of `.*`?

Reply:
The point of this test is to test out our logic, not to test the JDK `Matcher` logic, so testing with a well-known regex (such as `.*`) is appropriate in terms of test coverage.